packages/orchestration/src/exos/ica-account-kit.js

```diff
@@ -133,20 +142,30 @@ export const prepareIcaAccountKit = (zone, { watch, asVow }) =>
          * Submit a transaction on behalf of the remote account for execution on
          * the remote chain.
          *
+         * Set `relativeTimeoutNs` to provide a timeout for the IBC packet.
+         *
+         * `TxBody` fields like `timeoutHeight` and `memo` can be set, but these
+         * typically do not affect IBC app protocols like PFM, ICA.
+         *
          * @param {AnyJson[]} msgs
-         * @param {Omit<TxBody, 'messages'>} [opts]
+         * @param {Partial<
+         *   Omit<TxBody, 'messages'> & { relativeTimeoutNs: bigint }
+         * >} [opts]
```

Comment:
Maybe the following would be clearer (given `@import { SendOptions } from '@agoric/network';`)

```suggestion
         * @param {Partial<
         *   Omit<TxBody, 'messages'> & { sendOpts?: SendOptions }
         * >} [opts]
```